src/modules/70-pipeline/pages/webhooks/WebhookEvents/WebhookEvents.tsx

```diff
@@ -199,6 +233,9 @@ export default function WebhookEvents(): JSX.Element {
             <Text color={Color.GREY_800} iconProps={{ size: 14 }}>
               {getString('total')}: {data?.pagination?.total}
             </Text>
+            <span className={css.reload}>
+              <Icon name="command-rollback" onClick={refetch as () => void} />
```

Comment:
Why we need to typeCast `refetch` if it is undefined then should not we use refetch || noop?

Reply:
Actually, you should consider rewriting this as neither `onClick` nor `refetch` have no arguments. With this `onClick` will pass `refetch` the event object when it's actually expecting options to be passed. Fix is pretty simple though
```suggestion
              <Icon name="command-rollback" onClick={() => refetch()} />
```

Reply:
If you see the above comment i actually started with this and ended here :P

Reply:
Passing the event into refetch isn't a great idea and personally I'm surprised it's not failing. If you don't want to pass an inline callback, use `useCallback` to stabilise the reference.

```typescript
const iconOnClick = useCallback(() => refetch(), [refetch])

// ...

<Icon name="command-rollback" onClick={iconOnClick} />
```